docs/concepts/registries.md

```diff
@@ -43,6 +43,37 @@ public void registerBlocks(RegistryEvent.Register<Block> event) {
 }
 ```
 
+### Registries that aren't Forge Registries
+
+Due to some peculiarities of vanilla code, not all registries are wrapped by Forge. One of the more common examples of this is with some worldgen registries, such as configured features and placed features.
+
+It is safe to use these registries in your mods when a forge variant isn't available. However, without the benefit of registry objects, we have to be careful of when we do the registration and how.
+
+Here is an example. Here, we just use vanilla's method `Registry#register` while being sure to populate it at a later date.
+
+```java
```

Comment:
Can you explain this without copypasta code? 

It's a giant pain for us receiving support requests, and having to maintain this snippet when the internals of vanilla code changes.